### PR TITLE
Enable kamon

### PR DESCRIPTION
**Pull request by wwajerowicz**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,7 +1,8 @@
+import sbt.Keys.name
```
Comment (bogdanromanx) [inline build.sbt]: this can be removed

File: modules/service/src/main/scala/ch/epfl/bluebrain/nexus/iam/service/Main.scala [hunk 2]
```diff
@@ -67,6 +70,7 @@ object Main {
     cluster.joinSeedNodes(seeds.toList)
 
     as.registerOnTermination {
+      Kamon.shutdown()
```
Comment (bogdanromanx) [inline modules/service/src/main/scala/ch/epfl/bluebrain/nexus/iam/service/Main.scala]: leave the cluster before stoping kamon

**Comment by wwajerowicz** [comment]
retest this please

**Comment by bogdanromanx** [comment]
retest this please

**Pull request merged**